examples/mnist.py

```diff
@@ -97,10 +97,11 @@ def create_iter_functions(dataset, output_layer,
     batch_slice = slice(
         batch_index * batch_size, (batch_index + 1) * batch_size)
 
-    objective = lasagne.objectives.Objective(output_layer, loss_function=lasagne.objectives.multinomial_nll)
+    objective = lasagne.objectives.Objective(output_layer,
+        loss_function=lasagne.objectives.categorical_crossentropy)
 
     loss_train = objective.get_loss(X_batch, target=y_batch)
-    loss_eval = objective.get_loss(X_batch, target=y_batch, deterministic=True)
+    loss_eval = objective.get_loss(X_batch, target=y_batch,deterministic=True)
```

Comment:
This one is still open (reinserting the blank). Sander can take care of this while merging.
@dnouri: I'm not a pep8 expert, but shouldn't this have been caught by Travis with your new setup?